hca2scea-backend/script.py

```diff
@@ -420,7 +420,7 @@ def first_letter(str):
         if project_details.get('related_experiment'):
 
             idf_file_contents = f"""\
```

Comment:
I would move this bit to its own file, as a template.
Jinja would be a good template engine. Read about it in https://jinja.palletsprojects.com/en/3.0.x/